src/lib/uTop_main.ml

```diff
@@ -1577,7 +1596,14 @@ let interact ?(search_path=[]) ?(build_dir="_build") ~unit ~loc:(fname, lnum, cn
     failwith "Couldn't find location in cmt file"
   with Found env ->
   try
-    List.iter Topdirs.dir_directory (search_path @ cmt_infos.cmt_loadpath);
+    let cmt_loadpath = 
+#if OCAML_VERSION >= (5, 2, 0)
+    cmt_infos.cmt_loadpath.visible
```

Comment:
Not sure if I should account for hidden paths here, didn't make sense to do so as to why I directly accessed field `visible` of the `paths` type.